manifests/controllerconfig.crd.yaml

```diff
@@ -441,6 +441,9 @@ spec:
                 uid:
                   description: 'UID of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids'
                   type: string
+            rendererVersion:
+              description: operator version
+              type: string
```

Comment:
can we just use the metadata.Annotations field instead of adding a new field to the spec? I don't remember why we need this in the past but it does make sense to leverage the already existing field instead of adding a new one